skiko/src/jvmMain/kotlin/org/jetbrains/skiko/FrameDispatcher.kt

```diff
@@ -0,0 +1,34 @@
+package org.jetbrains.skiko
+
+import kotlinx.coroutines.CompletableDeferred
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.yield
+import kotlin.coroutines.CoroutineContext
+
+/**
+ * Dispatch frame after call of [scheduleFrame]
+ */
+class FrameDispatcher(
```

Comment:
Is this the only file containing `kotlinx.coroutines` dependencies?

Reply:
There is also MacOsRedrawer/WindowsRedrawer/LinuxRedrawer